src/screens/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {PureComponent} from 'react';
+import {View, ScrollView} from 'react-native';
+
+import NotificationCard from './components/NotificationCard/NotificationCard';
+import ServicesNotificationCard from './components/ServicesNotificationCard/ServicesNotificationCard';
+
+import styles from './notificationsStyles';
+
+const arrNotifications = [
+	{
+		id: 1,
+		type: 'notification',
+		date: '12 Mar',
+		time: '04:00PM - 06:10PM',
+		operation: 'Scheduled',
+		temperature: '+22 C',
+		humidity: '34%',
+		weekly: true,
+		description: 'This is our first meeti in this room, so we will occupy it for a while.',
+	},
+	{
+		id: 2,
+		type: 'notification',
+		date: '11 Mar',
+		time: '05:00PM',
+		operation: 'Manually',
+		temperature: '+25 C',
+		humidity: '54%',
+		weekly: false,
+	},
+	{
+		id: 3,
+		type: 'servicesNotification',
+		date: '10 Mar',
+		time: '02:10PM',
+		message: 'Added new Cooler in the Room',
+		model: 'ASI-09HS4/ASO-09HS4',
+		serial: 'ad76aaa0-838f-4f0c-b94e-8c0d1d2e41eb',
+	},
+	{
+		id: 4,
+		type: 'notification',
+		date: '08 Mar',
+		time: '9:00PM',
+		operation: 'Manually',
+		temperature: '+21 C',
+		humidity: '50%',
+		weekly: true,
+		description: 'This is our first meeti in this room, so we will occupy it for a while.',
+	},
+	{
+		id: 5,
+		type: 'servicesNotification',
+		date: '07 Mar',
+		time: '04:00PM',
+		message: 'Added new Cooler in the Room',
+		model: 'ASI-09HS4/ASO-09HS4',
+		serial: 'ad76aaa0-838f-4f0c-b94e-8c0d1d2e41eb',
+	},
+];
+
+class Notifications extends PureComponent {
+	public renderContent = (notification: any) => {
+		switch (notification.type) {
+			case 'notification': {
+				return <NotificationCard notification={notification} key={notification.id} />;
+			}
+			case 'servicesNotification': {
+				return <ServicesNotificationCard notification={notification} key={notification.id} />;
+			}
+		}
+	}
+
+	public render() {
+		return (
+			<ScrollView style={styles.container}>
+				<View style={styles.border} />
+				{
+					arrNotifications.map(notification => this.renderContent(notification))
```

Comment:
arrNotifications.map(this.renderContent)

Reply:
done